apps/bajour/next.config.js

```diff
@@ -43,7 +43,57 @@ const nextConfig = {
       }
     }
   },
-  transpilePackages: ['@wepublish/ui', '@wepublish/website', 'react-tweet']
+  transpilePackages: ['@wepublish/ui', '@wepublish/website', 'react-tweet'],
+  async redirects() {
+    return [
+      {
+        source: '/home',
+        destination: '/',
+        permanent: false
+      },
+      {
+        source: '/a/:id/:slug',
+        destination: '/a/:slug',
+        permanent: false
+      },
+      {
+        source: '/tag/:slug',
+        destination: '/a/tag/:slug',
+        permanent: false
+      },
+      {
+        source: '/profile/dashboard',
+        destination: '/login',
+        permanent: false,
+        has: [
+          {
+            type: 'query',
+            key: 'jwt'
+          }
+        ]
+      },
+      {
+        source: '/profile/dashboard',
+        destination: '/profile',
+        permanent: false
+      },
+      {
+        source: '/member-uebersicht',
```

Comment:
Those 3 probably should be split into 3 pages to stay that way, right @michael-scheurer ?